### PR TITLE
`watchFragment` fixes

### PR DESCRIPTION
**Pull request by alessbell**
Glad I started implementing the integration with `apollo-angular` because I found some minor issues :)

### CODE REVIEW

File: src/core/ApolloClient.ts [hunk 1]
```diff
@@ -239,6 +239,7 @@ export class ApolloClient<TCacheShape> implements DataProxy {
     this.watchQuery = this.watchQuery.bind(this);
     this.query = this.query.bind(this);
     this.mutate = this.mutate.bind(this);
+    this.watchQuery = this.watchQuery.bind(this);
```
Comment (jerelmiller) [inline src/core/ApolloClient.ts:242]:
```suggestion
    this.watchFragment = this.watchFragment.bind(this);
```

Did you mean to write this? I think this was your intention based on the changelog 🙂

**Review by jerelmiller — APPROVED** [review]
I trust you'll update the constructor as you see fit 🙂. Good catch!

File: src/core/ApolloClient.ts (hunk 1 above)
Comment (alessbell) [reply src/core/ApolloClient.ts:242]: So weird! Yes, I definitely meant `watchFragment` there 🥲

**Pull request merged**